Modules/xASL_module_BIDS2Legacy.m

```diff
@@ -68,8 +67,6 @@
         %% 1. Get subjectID & sessionID from x.modules.bids2legacy.BIDS
 
         %   !!!!! NOTE THAT SESSION (BIDS) IS A VISIT (LEGACY) HERE, NOT A RUN (BIDS) !!!!!
```

Comment:
In this function we use BIDS and legacy nomenclature interchanged, which is understandable because we do the conversion. But I would use BIDS nomenclature throughout in this function. We probably don't need the legacy nomenclature (sub-001_20240201 we can still call subjectSession).